### ISSUE TITLE
Error when using CUDA for transcription

### ISSUE THREAD

**Issue by ndrewpj** [rishikanthc/Scriberr#217]
I have Ubuntu 24.04 with 4 GPU RTX 1080, Ollama configured.

When using CPU - all is fine, but when using CUDA - I got the error below. Important notice - the LLM is loaded to the VRAM for some second then it is unloaded - this proofs that the GPUs are visible, CUDA is installed  correctly (I am using this GPU cluster in other tool) as well as the model. The VRAM is 32GB and not used.

>>Performing transcription...

/app/whisperx-env/WhisperX/.venv/lib/python3.11/site-packages/pyannote/audio/utils/reproducibility.py:74: ReproducibilityWarning: TensorFloat-32 (TF32) has been disabled as it might lead to reproducibility issues and lower accuracy.

It can be re-enabled by calling

   >>> import torch

   >>> torch.backends.cuda.matmul.allow_tf32 = True

   >>> torch.backends.cudnn.allow_tf32 = True

See https://github.com/pyannote/pyannote-audio/issues/1370 for more details.

  warnings.warn(

Unable to load any of {libcudnn_cnn.so.9.1.0, libcudnn_cnn.so.9.1, libcudnn_cnn.so.9, libcudnn_cnn.so}

**Comment by rishikanthc** [comment]
This is a known issue.. check #152 in the meantime to fix this.. the changes have been incorporated into main but the new docker image hasn't been published..

**Comment by rishikanthc** [comment]
The cuda issues have been resolved and you should be able to run it fine now. Closing this issue. Feel free to reopen if you hit the same issue.
